src/app/pages/tabs/profile/profile.page.ts

```diff
@@ -54,14 +56,23 @@ export class ProfilePage implements OnInit {
 
   async fetchProfileDetails() {
     var response = await this.profileService.profileDetails();
-    console.log(response);
-    this.formData.data = _.get(response, 'result');
-    if (this.formData.data.name) {
+    this.formData.data = _.get(response, 'user');
+    if (this.formData?.data?.about) {
       // TODO: remove the below line later
       this.showProfileDetails = true;
     }
   }
 
+  async doRefresh(event){
+    var res = await this.profileService.getProfileDetailsAPI();
+    this.formData.data = _.get(res, 'result');
```

Comment:
return without result from the service,

Reply:
Else everywhere you need to write like this

Reply:
i didnt understand.